docs/reference/licenses.nuget.org.md

```diff
@@ -0,0 +1,59 @@
+# licenses.nuget.org
+
+## Rationale
+
+With introduction of the [license expressions](nuspec.md#license) a requirement to have a reliable service that would
+provide a reference text for individual license identifiers, exception identifiers or license expression emerged.
+The one with a stable URL schema, that is not susceptible to the link rot, so that we can safely use it to provide
+backwards compatibility for older clients.
+
+[`licenses.nuget.org`](https://licenses.nuget.org) fulfills that role. Gallery uses it to provide the license
+text reference for the pacakages that specify their license using license expression and `nuget.exe pack` operation
```

Comment:
`the pacakages -> packages` (spelling and remove the article)

Reply:
Make the `nuget.exe pack` part a new sentence.

Reply:
Rewrote the `nuget.exe pack` portion a bit to be more correct.